rdf_differ/config.py

```diff
@@ -0,0 +1,15 @@
+"""
+config.py
+Date: 20/07/2020
+Author: Mihai Coșleț
+Email: coslet.mihai@gmail.com
+"""
+import os
+
+
+def get_envs() -> dict:
```

Comment:
`file.rdf` is a fallback, if it was not defined it uses this one. 

yeah, agree that we need to set the file depending on the type of the files used by the user. does this imply that we also have to check that both files used by the user are the same file type? 

I don't understand the last statement:
> We could try to rely on the original filenames provided by the user. This can be enforced that, the API requires the filenames as a parameter along with the file content.

Reply:
Maybe just fail hard? Not provided `DEFAULT_FILENAME` no party?